common/build/eslint-plugin-fluid/src/test/enforce-no-unchecked-record-access/enforce-no-unchecked-record-access.test.js

```diff
@@ -40,67 +40,68 @@ describe("ESLint Rule Tests", function () {
 	it("Should report errors for unchecked record access for indexed record of strings", async function () {
 		const result = await lintFile("indexedRecordOfStrings.ts");
 
-		assert.strictEqual(result.errorCount, 10, "Should have 10 errors");
+		assert.strictEqual(result.errorCount, 11, "Should have 11 errors");
 
 		assert.strictEqual(
 			result.messages[0].message,
 			"'indexedRecordOfStrings.a' is possibly 'undefined'",
 		);
-		assert.strictEqual(result.messages[0].line, 21);
+		assert.strictEqual(result.messages[0].line, 25);
```

Comment:
nit - It's generally good practice to avoid "magic numbers". Maybe it would be better to have something like

```ts
const expectedLines = [25, 27, 28, 29, 46, ...];
```